model/accounts_mgmt/v1/one_metric_type.model

```diff
@@ -0,0 +1,44 @@
+/*
+Copyright (c) 2021 Red Hat, Inc.
+
+Licensed under the Apache License, Version 2.0 (the "License");
+you may not use this file except in compliance with the License.
+You may obtain a copy of the License at
+
+  http://www.apache.org/licenses/LICENSE-2.0
+
+Unless required by applicable law or agreed to in writing, software
+distributed under the License is distributed on an "AS IS" BASIS,
+WITHOUT WARRANTIES OR CONDITIONS OF ANY KIND, either express or implied.
+See the License for the specific language governing permissions and
+limitations under the License.
+*/
+
+// healthy|unhealthy|unknown
+enum HealthStateEnum {
```

Comment:
The values of the enum should be inside:

```
enum HealthState {
        Healthy
        Unhealthy
        Unknown
}
```

Also no need for the `Enum` suffix in the name of the type.

Reply:
If we already on enums. PTAL at model/accounts_mgmt/v1/subscription_type.model There are a lot of enumerators, but their values are not permitted by current compiler. Is there some way to fix it?

Reply:
Not sure what compiler are you referring to. Please elaborate.